src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -283,5 +282,20 @@ public override Uri GetUri(string name)
         {
             return new Uri(_directory.Uri, name);
         }
+
+        public override async Task<bool> AreSyncronized(Uri sourceUri, Uri destinationUri)
```

Comment:
These aren't really a `sourceUri` and a `destinationUri`. For Catalog2Dnx, you're copying one to the other, but outside of that context, this method is identical regardless of which is which.

IMO something like `Uri firstResourceUri` and `Uri secondResourceUri` would make more sense.

Reply:
Updated with the suggested names.